recipes/pydenticon/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pydenticon" %}
+{% set version = "0.3.1" %}
+{% set python_min = "3.6" %}
```

Comment:
conda-forge does not officially support python < 3.9. You can just remove this definition of `python_min` here and the recipe will pick up the global pin.